tests/dags/providers/provider_api_scripts/resources/JsonLoad.py

```diff
@@ -0,0 +1,8 @@
+import json
+from pathlib import Path
+
+
+def get_resource_json(folder_name, json_name):
```

Comment:
One thing I'm seeing in a lot of the files is that we're still retaining a `_get_resource_json` function which merely calls this function with the resource folder. This might be a bit tricky, but one way we could improve this would be to create a higher-order function which _generates_ another function. This could look something like:

```python
import json
from pathlib import Path
from typing import Callable


def make_resource_json_func(folder_name: str) -> Callable[[str], Path]:
    resources = Path(__file__).parent / folder_name

    def get_resource_json(resource_name: str) -> Path:
        return json.loads((resources / resource_name).read_text())

    return get_resource_json
```

Then, in the actual test functions, we can replace the `_get_resource_json` declaration with something like this:

```python
_get_resource_json = make_resource_json_func("brooklynmuseum")
```

That would make `_get_resource_json` within a given testing module behave _exactly_ as it had before without any additional test changes, _and_ we get to remove the `_get_resource_json` declarations in each of the modules too! If that change makes sense (happy to describe it a bit more too), do you think you'd be willing to make that for this PR?

Reply:
I need a mind blown reaction lol! Yes this makes perfect sense and is the type of solution I was reaching for when I was implementing. I'll work on that this weekend. 

Are type hints encouraged for this project? I usually use them but I didn't notice them in the files I was working on so I avoided them.

Reply:
Yes, @Wbrown633, we encourage the use of type hints!
You can see a good example in `openverse_catalog/dags/providers/provider_api_scripts/provider_data_ingester.py`

Reply:
Fantastic! I'm glad that made sense, feel free to ask us if you have any questions 😄 

Type hints are definitely encouraged, and that's the nice thing about Python - while much of this code hasn't been touched in a bit, we can add type hints to the code we _do_ add/modify and don't need to have 100% coverage if we start adding it 🙂 in other words, yes please!